### PR TITLE
MRG Edit docs

### PR DESCRIPTION
**Pull request by teonlamont**
The doc strings had incorrect information for finding ECG automatically, and some wrong labeling.

also, these docs have no author labels or license! could someone let me know who wrote them or send my branch a pull request

### CODE REVIEW

**Comment by dengemann** [comment]
thanks Teon!!!

File: mne/preprocessing/ecg.py [hunk 1]
```diff
@@ -221,8 +222,9 @@ def create_ecg_epochs(raw, ch_name=None, event_id=999, picks=None,
         The raw data
     ch_name : str
```
Comment (Eric89GXL) [inline mne/preprocessing/ecg.py]: If `None` is an option then it should be listed here, too

**Comment by dengemann** [comment]
Look at git blame :) Alex wrote a lot in this file, I've added the ecg /eog custom functions and the surrogate ECG via MEG.

**Comment by dengemann** [comment]
And as usual, Eric corrected and optimized it up and down.

**Comment by teonlamont** [comment]
Should we standardized the license line, we have a variety: Simplified BSD, BSD (3-clause)?

**Comment by Eric89GXL** [comment]
Also, do we need an "authors" list at the top of each file at all? We aren't vigilant about having them up-to-date...

**Comment by teonlamont** [comment]
@agramfort always suggests them. I just thought that was de facto rule.

**Comment by Eric89GXL** [comment]
Yeah, but I have a feeling if we compared `git blame` to the author list at the top of the file there would be many discrepancies. In fact, if we want to keep it, I could write a quick script to use `git blame` to enumerate the authors for each file in the repo, and add it to the `Makefile` :)

**Comment by agramfort** [comment]
let's keep things simple. You add your name if you made a real contribution to
a file: new function, logic refactoring, new options.

the idea is also to know who to ask if there is a bug in a file.

so let's keep the name of authors on top of the files and do it manually.

**Comment by agramfort** [comment]
thx @teonlamont !

**Pull request merged**